scripts/integrations/main.py

```diff
@@ -0,0 +1,89 @@
+import typer
+import configparser
+import json
+import os
+import sys
+import shutil
+from rich import print
+
+app = typer.Typer()
+base_integration_path = "../integrations"
+dirs = ["assets", "info", "samples", "schema", "test"]
+
+
+def createIntegrationDirs(directory, force):
+    if os.path.isdir(directory):
+        if force:
+            force = typer.confirm(f"Are you sure you want to delete {directory}?")
+            if force:
+                shutil.rmtree(directory)
+                os.makedirs(directory)
+                for dir in dirs:
+                    os.makedirs(directory + "/" + dir)
+        else:
+            print(f"[red]Directory {directory} already exists[/red]")
+            sys.exit(1)
+    else:
+        os.makedirs(directory)
+        for dir in dirs:
+            os.makedirs(directory + "/" + dir)
+
+
+def populateDefaultVars(var_name):
+    config = configparser.ConfigParser()
+    config.read('config.ini')
+    return (config.get("DEFAULT", var_name))
+
+
+def generateJson(directory, integration_name, integration_desc, schema, license):
+    data = {
+        "name": integration_name,
+        "version": {
+            "integ": "0.1.0",
+            "schema": schema,
+            "resource": "^1.23.0",
+            "license": license
+        },
+        "description": integration_desc,
+        "identification": "instrumentationScope.attributes.identification",
+        "categories": [],
+        "collection": [{}],
+        "repo": {
+            "github": "https://github.com/opensearch-project/observability/tree/main/integrarions/"+integration_name
+        }}
+
+    with open(directory + '/config.json', 'w', encoding='utf-8') as f:
+        json.dump(data, f, ensure_ascii=False, indent=4)
```

Comment:
This would only be for local data dump - should we think about how can we add this globally, not just on this cluster? Automated way to create a PR, etc.?

Reply:
@derek-ho 
Is that a design question for the team or something for me to rethink about?
ain't sure :)

Reply:
see [design](https://github.com/opensearch-project/observability/issues/1451) RFC